packages/vite/package.json

```diff
@@ -11,33 +12,36 @@
   "exports": {
     "./package.json": "./package.json",
     ".": {
-      "types": "./dist/index.d.ts",
-      "default": "./cjsWrapper.js"
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      },
+      "require": {
+        "types": "./dist/cjs/index.d.ts",
+        "default": "./dist/cjs/index.js"
+      }
```

Comment:
Should we just let TSC find the types here? Instead of being explicit about it? I *think* I prefer being explicit. But I can (easily) be convinced otherwise.

Reply:
I confirmed that without this expressive syntax, attw will complain about ESM masqurading types. This is the syntax we should stick with!